airflow/providers/cncf/kubernetes/triggers/pod.py

```diff
@@ -254,6 +254,7 @@ async def _wait_for_container_completion(self) -> TriggerEvent:
                         "message": "Container state failed",
                     }
                 )
+            self.log.info("Container is not completed and still working.")
```

Comment:
How often this log entry is registered?

Reply:
After every self.poll_interval seconds. The default value is 2 seconds.